### ISSUE TITLE
Superlatives

### ISSUE THREAD

**Issue by aryamanarora** [carmls/snacs-guidelines#139]
In Hindi, the superlative form of an adjective uses the following construction:

> सब**से** अच्छा
> _sab-**se** acchā_
> all.**ABL** good
> "best"

The comparative degree + "than" is indicated with the same case marker (being attached to the thing that is compared to), and that sense we've agreed on labelling ComparisonRef\~ComparisonRef since it's a very prototypical use. Here it's literally "better than all".

The problem with the superlative is that it can be construed as both a comparison (ComparisonRef) or picking out a single item from a set (Whole).

Which one makes more sense? The sentence we were discussing:

> दुनिया में मेरा सबसे प्रिय दोस्त
> _duniyā mẽ merā sabse priyă dost_
> world LOC 1S.GEN all.ABL dear friend
> "my dearest friend in the world"

Options we narrowed down to:

my dearest<sub>ComparisonRef</sub> friend in<sub>Whole\~Locus</sub> the world
my dearest<sub>Whole</sub> friend in<sub>Locus</sub> the world
?my dearest<sub>Whole</sub> friend in<sub>Whole\~Locus</sub> the world

**Comment by aryamanarora** [comment]
I think सबसे is sufficiently grammaticalized that it doesn't necessarily mean a set anymore. So ComparisonRef is good.

**Comment by nitinvwaran** [comment]
TL;DR: ComparisonRef for सबसे, and Whole~Source for  में , leading to:
> my dearest (ComparisonRef) friend in (Whole~Source) the world

As this could be the equivalent of an 'out_of' construction on a Set.

For Hindi, looking at Omkar N Koul's book _Modern Hindi _Grammar__ , he describes uses of से (se) (page 47 - 51 in the 2008 version), and one of them is "difference or comparison in quality and quantity". He does not describe a usage that includes Set-Relationship concepts , like Whole. 

Based on that, सबसे (dearest) might be ComparisonRef if ComparisonRef vs Whole.

Then there's another sentence from Omkar Koul again, which is structurally very similar to the sentence in the Little Prince, and he describes the usage of में (mẽ) too as a "Comparison", (and he has another specific section for Location with other examples)

Little Prince:
> दुनिया में मेरा सबसे प्रिय दोस्त
> duniyā mẽ merā sabse priyă dost
> world LOC 1S.GEN all.ABL dear friend
> "my dearest friend in the world"

Omkar Koul:
> इन लड़कों में अमित सब से चास्तु हे
> in ladkon mẽ amit sab se cust he
> these boys-obl in Amit all from active are
> Amit is the most active out of all these boys.

The 'out of' might suggest Whole~Source for में (from en guidelines 365.d), which I hadn't considered before. (The 'out of' in the example above is Omkar Koul's translation). There is also a 'me se' 'में से' construction in Hindi which could literally be out_of?

'Out of' kind of makes sense to me based on these examples too (rough literal translations are mine, hindi sentences from Sketch Engine's Hindi Web 2012 corpus):

> क्युकी फॅमिली मैं सब से	बड़ा लड़का मैं (ही) हूँ
> because family mẽ all from big boy i (emphasis) am 
> because I am the biggest boy [in / out of all] the family 
> 
> मैं तुम से इस दुनिया में सब से ज़्यादा प्यार करती हू
> I you se this world mẽ all from more love do 
> I love you the most out of everyone in the world 
> The first se is not relevant to this discussion
